Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -39,9 +39,15 @@ public async Task InitDefaultCustomCommands()
             {
                 customCommands.AddRange(new List<AddCustomCommandsCommand>
                 {
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
-                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<Roles>[\w\W\s\""]*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<Roles>.*)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<Roles>.*)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<Roles>.*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Responses.BotCommands.AddResponseCommand", @"-add\s*response\s*-onevent\s*(?<OnEvent>.*)\s*-message\s*(?<Message>.*)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Responses.BotCommands.UpdateResponseCommand", @"-update\s*response\s*-onevent\s*(?<OnEvent>.*)\s*-message\s*(?<Message>.*)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Responses.BotCommands.RemoveResponseCommand", @"-remove\s*response\s*-onevent\s*(?<OnEvent>.*)", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Protection.BotCommands.MuteCommand", @"-mute\s*(?<User>.*)\s*-t\s*(?<Time>.*)\s*-reason\s*(?<Reason>.*)", serverId),
```

Comment:
`(?<User>.*)`
dlaczego pozwalasz na wpisanie tu czegokolwiek?

Reply:
Dzięki temu zamiast braku reakcji ze strony bota mamy komunikat, że albo są niepoprawne argumenty albo, że jest nie wystarczająca liczba argumentów.

Wiem, że w przyszłości będą helpy do komend, ale niech na razie tak będzie, poza tym do tych helpów też może by to było potrzebne.

Reply:
`Wiem, że w przyszłości będą helpy do komend`
no w tej przyszłości byśmy nie chcieli przepisywać wszystkich regexów od zera
bo praca z nimi jest niezbyt wygodna


te regexy to coś co konfiguruje sobie właściciel konkretnego serwera, to nie jest coś co ma być wbudowane w bota
na razie trzymamy to w kodzie, bo nie mamy opcji konfigurowania tego z zewnątrz 


trzeba o tym myśleć tak, jakby to było coś zewnętrznego a nie wewnętrznego
typowy właściciel serwera spróbowałby zabezpieczyć komende na poziomie regexa